snuba/query/parser/conditions.py

```diff
@@ -23,7 +23,7 @@ class InvalidConditionException(Exception):
 
 
 def parse_conditions(
-    operand_builder: Callable[[Any], TExpression],
+    operand_builder: Callable[[Any, Optional[Any], Optional[str]], TExpression],
```

Comment:
why is the second argument to operand_builder `Optional[Any]` ? I think it should be ColumnSet